lib/mix/tasks/tidewave.install.ex

```diff
@@ -0,0 +1,142 @@
+defmodule Mix.Tasks.Tidewave.Install.Docs do
+  @moduledoc false
+
+  def short_doc do
+    "Installs `tidewave` into your project"
+  end
+
+  def example do
+    "mix igniter.install tidewave"
+  end
+
+  def long_doc do
+    """
+    #{short_doc()}
+
+    ## Example
+
+    ```bash
+    #{example()}
+    ```
+    """
+  end
+end
+
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.Tidewave.Install do
+    @shortdoc "#{__MODULE__.Docs.short_doc()}"
+
+    @moduledoc __MODULE__.Docs.long_doc()
+    @plug_example """
+        if code_reloading? do
+          socket "/phoenix/live_reload/socket", Phoenix.LiveReloader.Socket
+        +  plug Tidewave
+          plug Phoenix.LiveReloader
+          plug Phoenix.CodeReloader
+          plug Phoenix.Ecto.CheckRepoStatus, otp_app: :my_app
+        end
+    """
+
+    use Igniter.Mix.Task
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        group: :tidewave,
+        example: __MODULE__.Docs.example()
+      }
+    end
+
+    @impl Igniter.Mix.Task
+    def igniter(igniter) do
+      igniter
+      |> setup_phoenix()
+      |> Igniter.add_notice("""
+      Tidewave next steps:
+
+      * Set up your editor to integrate with your new Tidewave MCP: https://github.com/tidewave-ai/tidewave_phoenix/tree/main/pages/editors
+      * Configure the tidewave plug: https://github.com/tidewave-ai/tidewave_phoenix/tree/main?tab=readme-ov-file#configuration
+      """)
+    end
+
+    defp setup_phoenix(igniter) do
+      {igniter, endpoint} =
+        Igniter.Libs.Phoenix.select_endpoint(
+          igniter,
+          nil,
+          "Which endpoint should serve your tidewave MCP?"
+        )
+
+      if endpoint do
+        add_plug_to_endpoint(igniter, endpoint)
+      else
+        Igniter.add_warning(igniter, """
+        No endpoint found or selected for tidewave setup. Please add the plug manually, for example:
+
+        #{@plug_example}
+        """)
+      end
+    end
+
+    defp add_plug_to_endpoint(igniter, endpoint) do
+      Igniter.Project.Module.find_and_update_module!(igniter, endpoint, fn zipper ->
+        with {:ok, zipper} <- Igniter.Code.Common.move_to(zipper, &code_reloading?/1),
```

Comment:
we changed this in https://github.com/tidewave-ai/tidewave_phoenix/pull/27

Reply:
Perfect, will update later today 😄